cilium/datadog_checks/cilium/metrics.py

```diff
@@ -1,6 +1,12 @@
 # (C) Datadog, Inc. 2019-present
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+from copy import deepcopy
+
+AGENT_V2_OVERRIDE = {
+    'cilium_endpoint_regenerations': 'endpoint.regenerations',
```

Comment:
I'm trying to understand why these two metrics are being overridden. Is it because they are counters but don't follow the OpenMetrics counter spec (i.e. they don't have a `_total` suffix)?

Reply:
These overrides are to prevent the check from removing parts of the metric due to the generate_metrics function. I was trying to avoid 2 different metrics maps constants